cmd/arc_summary/arc_summary.py

```diff
@@ -31,6 +31,10 @@
 #
 # If you are having troubles when using this script from cron(8) please try
 # adjusting your PATH before reporting problems.
+# 
```

Comment:
./cmd/arc_summary/arc_summary.py:34:2: W291 trailing whitespace

Reply:
@gmelikov Argh, missed that one. Sorry, next try.

Say, let's assume for the sake of an argument that somebody wanted to create a version of arc_summary.py explicitly for Python 3 (creatively named "arc_summary3.py" maybe) so we could completely get rid of all that legacy code, to be included in the same folder along with this version of the program. How would you go about adding this to GitHub? And since it would break the current Python tests (CentOS 6, I'm looking at you), is this even possible?